src/server/i18n/cy.json

```diff
@@ -170,5 +178,64 @@
     "pending_notice": "Mae taliad yn yr arfaeth ar gyfer y cod talu hwn ac nid yw'r botwm hwn ar gael ar hyn o bryd. Arhoswch hyd at MINUTES munud i’r taliad gael ei brosesu neu efallai y codir tâl arnoch ddwywaith.",
     "go_back": "Dychwelyd",
     "pay_button": "Parhau"
+  },
+  "cookie_preferences_page": {
+    "title": "Cookies on Roadside Payments service",
```

Comment:
No translations for this page yet?

Reply:
No translations as yet. They may be introduced at a later date.